model/src/main/proto/topic.proto

```diff
@@ -249,3 +252,35 @@ message CellularDataPreference {
   // Preference on whether to use cellular data
   bool use_cellular_data = 2;
 }
+
+// Corresponds to a single subtopic that can be trained, or reviewed.
```

Comment:
Update: 
`// Corresponds to a single subtopic.`

let's not put any restriction regarding trained and reviewed.

Reply:
Updated.